library/std/src/os/fd/owned.rs

```diff
@@ -24,9 +24,14 @@ use crate::sys_common::{AsInner, FromInner, IntoInner};
 /// passed as an argument, it is not captured or consumed, and it never has the
 /// value `-1`.
 ///
-/// This type's `.to_owned()` implementation returns another `BorrowedFd`
-/// rather than an `OwnedFd`. It just makes a trivial copy of the raw file
-/// descriptor, which is then borrowed under the same lifetime.
+/// This type does not have a [`ToOwned`][crate::borrow::ToOwned]
+/// implementation. Calling `.to_owned()` on a variable of this type will call
+/// it on `&BorrowedFd` and use the `Clone::clone()` like `ToOwned` does for
```

Comment:
```suggestion
/// it on `&BorrowedFd` and use `Clone::clone` like `ToOwned` does for
```

OR

```suggestion
/// it on `&BorrowedFd`, thus will `Clone::clone(&fd)` like `ToOwned` does for
```

Reply:
to be more precise: it's the "the `Clone::clone()`" that is the problem insofar as it is a problem.

Reply:
Fixed.